android/src/test/java/com/amplitude/android/AmplitudeSessionTest.kt

```diff
@@ -64,7 +64,7 @@ class AmplitudeSessionTest {
         amplitudeDispatcherField.set(amplitude, dispatcher)
     }
 
-    private fun createConfiguration(storageProvider: StorageProvider? = null): Configuration {
```

Comment:
Should we keep the old test for backward compatibility and create new tests around auto capture for the new features?

Reply:
The old tests don't work since we're discarding `defaultTracking` changes after initialization. I can try to duplicate the configuration setup to test for `defaultTracking` and `trackingSessionEvents`.

Reply:
The old test shouldn't work. But it's still testing the default event tracking, so I think we shouldn't remove those test. Instead, we need to re-initialize Amplitude in every single test.

Reply:
Are those tests for testing the default event tracking behavior or test the set config after initialization? If it's for the former reason, then we shouldn't remove those.

Reply:
That test was for testing the default event tracking. I added it back in fc3ac6f1384c2ab0402549d419c73173ea96097c.